examples/experimental/ProductionMonitoring/stream_table_api.ipynb

```diff
@@ -0,0 +1,170 @@
+    "\n",
+    "**Please login to W&B and set your WB_ENTITY** before running this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51c30e7a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "from weave.monitoring import StreamTable"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5684f0b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = \n",
+    "WB_PROJECT = \"mesa\"\n",
+    "STREAM_TABLE_NAME = \"my_stream_table\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f676d96",
+   "metadata": {},
+   "source": [
+    "## Step 1: Define a StreamTable\n",
+    "\n",
+    "StreamTable has a single required argument: the name of the StreamTable object.\n",
+    "\n",
+    "```python\n",
+    "st = StreamTable(\"stacey/mesa/my_stream_table\")\n",
+    "```\n",
+    "\n",
+    "This takes the form `my_wb_entity/my_wb_project_name/my_stream_table_name` where you can modify the component names to the relevant strings (e.g. your W&B username or shared W&B team name, a new or existing W&B project name).\n",
+    "\n",
+    "**Note**: if you are logged in on the current machine and have a ~/.netrc file saved with your W&B credentials, this will be the default entity used if you do not pass in an entity string component to the StreamTable constructor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25bfa9dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "st = StreamTable(f\"{WB_ENTITY}/{WB_PROJECT}/{STREAM_TABLE_NAME}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "073d25c9",
+   "metadata": {},
+   "source": [
+    "## Step 2: Log some data\n",
+    "\n",
+    "To add rows to the StreamTable, call `.log()` on the StreamTable object. \n",
+    "`.log()` accepts a single dictionary or a list of dictionaries, where each dictionary entry corresponds to one row of the table. In each dictionary, the keys are column names and the values are the corresponding cell values.\n",
+    "\n",
+    "```python\n",
+    "st.log({\"one_column_name\" : \"value_a\", \"another_column_name\" : 7})\n",
+    "st.log([{\"one_column_name\" : \"value_b\", \"another_column_name\" : 19},\n",
+    "        {\"one_column_name\" : \"value_c\", \"another_column_name\" : 28},\n",
+    "        {\"one_column_name\" : \"value_d\", \"another_column_name\" : 36}]\n",
+    "```\n",
+    "\n",
+    "The first call to `.log()` will return a Weave Panel URL, where you can view, edit, and save the resulting StreamTable as a Weave Board, of the form:\n",
+    "\n",
+    "View data at: https://weave.wandb.ai/?exp=get%28%0A++++%22wandb-artifact%3A%2F%2F%2Fstacey%2Fmesa%2Fmy_stream_table%3Alatest%2Fobj%22%29%0A++.rows\n",
+    "\n",
+    "Subsequent log calls will silently append these rows to the StreamTable instance.\n",
```

Comment:
nit: drop the `silently` as it has a slightly negative connotation on the system not doing something right imo

Reply:
done/attempted! the weirdness to me as a user here is that I only get the "View data at" url on the FIRST log call, and then I can't really trigger it again, unless I manage to keep a reference to the original StreamTable variable, or if I re-declare it? Both patterns are a little strange—I want to have a way to specifically "view the current state of the StreamTable" without necessarily having to refind it/log data to it....

Reply:
I think longer term, we can configure the logging here. But in general, you are not going to be logging stream tables from notebooks, it will be in your production code so in app discovery of the table is more important than a console log line

Reply:
understood—and not sure how this will work long term for actual "in app" discovery, especially of Panels vs Boards.

Right now, given the name of a StreamTable object, I want to view its contents. I can
1) go to weave.wandb.ai > project name > table name
2) try to remember the name of a Board that uses that StreamTable object, since all Boards are saved under weave/